aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -82,9 +82,10 @@ class AwsFluentClientDecorator : RustCodegenDecorator {
             AwsFluentClientExtensions(types).render(writer)
         }
         val awsHyper = "aws-hyper"
-        rustCrate.mergeFeature(Feature("client", default = true, listOf(awsHyper, "aws-smithy-client")))
-        rustCrate.mergeFeature(Feature("rustls", default = true, listOf("$awsHyper/rustls")))
-        rustCrate.mergeFeature(Feature("native-tls", default = false, listOf("$awsHyper/native-tls")))
+        val awsSmithyClient = "aws-smithy-client"
+        rustCrate.mergeFeature(Feature("client", default = true, listOf(awsSmithyClient, awsHyper)))
+        rustCrate.mergeFeature(Feature("rustls", default = false, listOf("$awsHyper/rustls", "$awsSmithyClient/rustls")))
```

Comment:
I think we probably still want to default to rustls on the service crates so that customers don't need to specify a feature list to use the SDK. They will be able to switch to native-tls by specifying `no-default-features`, and then adding `native-tls` and `client` explicitly.